code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -477,6 +515,14 @@
 	special_light_key = "service"
 	hat_offset = 0
 
+/obj/item/robot_module/butler/do_transform_animation()
+	var/mob/living/silicon/robot/R = loc
+	R.cut_overlays()
+	R.setDir(SOUTH)
+	flick("maximillion", R)

```

Comment:
service borgs have multiple possible skins, so this animation will not always be the correct one.

Reply:
I already attempted to have it use the one related to the module, but it wasn't functional, so a general one would work for now. Also happens to be the most flashy of 'em.

Reply:
it's really bad to have it use the most humaniod version and then have turn into what appears to be a mediborg.